src/store/reducers/transaction.js

```diff
@@ -7,7 +7,7 @@ const transaction = (state = {}, action) => {
     case actionTypes.transactionLoaded:
       return {
         votesName: state.votesName || {},
-        success: action.data && action.data.senderId,
+        senderId: action.data && action.data.senderId,
```

Comment:
Changing `success` to `senderId` in the reducer and it's unit test and not anywhere else in the application logic means one of these two options:
- `success` was not used anywhere, so it can be removed here.
- `success` was used somewhere and that somewhere is now broken.
Either way, there is a problem.